[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:69 ************ Test Forgot Password 003 Start ************
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:71 Navigating to the url
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:74 clicking on my account
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:76 clicking on login link
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:79 clicking on forgot password link
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:80 ************ Verifying Test Forgot Password 003 ************
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:83 E-Mail Address
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:85 *********** Test Forgot Password 003 Passed *************
[32mINFO    [0m Test_004_Forgot_Password:test_forgot_password.py:91 ************** End of Test Forgot Password 003 ********************
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C7AF9D96D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/8d217ab3ed714b8a3b2dec01ba3ffc4e
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C7AF9DA350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/8d217ab3ed714b8a3b2dec01ba3ffc4e
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C7AFA183E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/8d217ab3ed714b8a3b2dec01ba3ffc4e